app/Services/Patreon/Resources/Campaign.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace App\Services\Patreon\Resources;
+
+class Campaign extends Resource
+{
+    /** @var int */
+    public $id;
+
+    /** @var string */
+    public $name;
+
+    /** @var int */
+    public $pledgedSum;
+
+    /** @var \App\Services\Patreon\Resources\ResourceCollection  */
+    public $rewards;
```

Comment:
Do we need to display this somewhere? If not, remove it. For all these resources only keep what you actually need.

Reply:
They are being used when filtering out the rewards with an amount above €50

```php
        $rewards = $campaign->rewards->filter(function (Reward $reward) {
            return $reward->amount >= 5000;
        });
```